src/material-experimental/mdc-checkbox/checkbox.ts

```diff
@@ -388,4 +388,17 @@ export class MatCheckbox
       nativeCheckbox.nativeElement.indeterminate = value;
     }
   }
+
+  /**
+   *  Prevent click events that come from the `<label/>` element from bubbling. This prevents the
+   *  click handler on the host from triggering twice when clicking on the `<label/>` element. After
+   *  the click event on the `<label/>` propagates, the browsers dispatches click on the associated
+   *  `<input/>`. By preventing clicks on the label by bubbling, we ensure only one click event
+   *  bubbles when the label is clicked.
+   **/
+  protected preventBubblingFromLabel(event: MouseEvent) {
+    if (event.target instanceof HTMLElement && this._label.nativeElement.contains(event.target)) {
```

Comment:
The `instanceof` check is going to fail in a Node environment. I think that you can just null check the `event.target` and then cast it to an `HTMLElement` for the `contains` call.

Reply:
Node environment – does that means this won't work with server-side rendering? I didn't consider server-side rendering.

Reply:
Either server-side rendering or in a unit test run through Node.

Reply:
good catch, I'll change it to just the null check with `instaceof`